docs/spec/auth/vesting.md

```diff
@@ -1,34 +1,6 @@
 # Vesting
 
-<!-- TOC -->
-
-- [Vesting](#vesting)
-  - [Intro and Requirements](#intro-and-requirements)
-  - [Vesting Account Types](#vesting-account-types)
-  - [Vesting Account Specification](#vesting-account-specification)
-    - [Determining Vesting & Vested Amounts](#determining-vesting--vested-amounts)
-      - [Continuously Vesting Accounts](#continuously-vesting-accounts)
-      - [Delayed/Discrete Vesting Accounts](#delayeddiscrete-vesting-accounts)
-    - [Transferring/Sending](#transferringsending)
-      - [Continuously Vesting Accounts](#continuously-vesting-accounts-1)
-        - [Delayed/Discrete Vesting Accounts](#delayeddiscrete-vesting-accounts-1)
-        - [Keepers/Handlers](#keepershandlers)
-    - [Delegating](#delegating)
-      - [Continuously Vesting Accounts](#continuously-vesting-accounts-2)
-        - [Delayed/Discrete Vesting Accounts](#delayeddiscrete-vesting-accounts-2)
-        - [Keepers/Handlers](#keepershandlers-1)
-    - [Undelegating](#undelegating)
-      - [Continuously Vesting Accounts](#continuously-vesting-accounts-3)
-        - [Delayed/Discrete Vesting Accounts](#delayeddiscrete-vesting-accounts-3)
-        - [Keepers/Handlers](#keepershandlers-2)
-  - [Keepers & Handlers](#keepers--handlers)
-  - [Initializing at Genesis](#initializing-at-genesis)
-  - [Examples](#examples)
-    - [Simple](#simple)
-    - [Slashing](#slashing)
-  - [Glossary](#glossary)
-
-<!-- /TOC -->
+<!-- TOC -->autoauto- [Vesting](#vesting)auto  - [Intro and Requirements](#intro-and-requirements)auto  - [Vesting Account Types](#vesting-account-types)auto  - [Vesting Account Specification](#vesting-account-specification)auto    - [Determining Vesting & Vested Amounts](#determining-vesting--vested-amounts)auto      - [Continuously Vesting Accounts](#continuously-vesting-accounts)auto      - [Delayed/Discrete Vesting Accounts](#delayeddiscrete-vesting-accounts)auto    - [Transferring/Sending](#transferringsending)auto      - [Keepers/Handlers](#keepershandlers)auto    - [Delegating](#delegating)auto      - [Keepers/Handlers](#keepershandlers-1)auto    - [Undelegating](#undelegating)auto      - [Keepers/Handlers](#keepershandlers-2)auto  - [Keepers & Handlers](#keepers--handlers)auto  - [Initializing at Genesis](#initializing-at-genesis)auto  - [Examples](#examples)auto    - [Simple](#simple)auto    - [Slashing](#slashing)auto  - [Glossary](#glossary)autoauto<!-- /TOC -->
```

Comment:
This looks really weird when just reading the file (e.g. in `vim`) - is this necessary for some reason?

Reply:
indeed if you display it in `rich diff` it looks weird as well

Reply:
Woaaaaaa. I have zero idea what happened here or when? I will resolve this.

Reply:
Fixed.